src/main/kotlin/no/nav/familie/ks/sak/kjerne/brev/domene/maler/BrevDto.kt

```diff
@@ -21,6 +21,10 @@ interface BrevDataDto {
     fun toBrevString(): String = objectMapper.writeValueAsString(this)
 }
 
+interface VedtaksbrevSammensattKontrollsak : BrevDataDto {
+    val sammensattKontrollsakFritekst: String
+}
+
```

Comment:
Lurer på om denne + VedtakFellesfelterSammensattKontrollsak egentlig bør legges inn i en separat fil.

Reply:
Endte med å legge den i to egne filer, en for interface og en for data klassen, da jeg ikke følte det hadde veldig sterkt tilhørlighet til hverandre (dvs. de bruker ikke hverandre)